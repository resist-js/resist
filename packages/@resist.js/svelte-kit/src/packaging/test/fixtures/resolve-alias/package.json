{
	"name": "resolve-alias",
	"version": "1.0.0",
	"description": "package using $lib alias"
}
