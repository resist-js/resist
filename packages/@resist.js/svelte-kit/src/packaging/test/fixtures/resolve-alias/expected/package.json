{
	"name": "resolve-alias",
	"version": "1.0.0",
	"description": "package using $lib alias",
	"type": "module",
	"exports": {
		"./package.json": "./package.json",
		"./Test.svelte": "./Test.svelte",
		".": "./index.js",
		"./baz": "./baz.js",
		"./sub/bar": "./sub/bar.js",
		"./sub/foo": "./sub/foo.js"
	},
	"svelte": "./index.js"
}
